### PR TITLE
Add queryTerrainElevation

### LINKED ISSUE #1654: bad position of 3D model when 3D terrain is enabled
The position of the 3D model and the map are not synchronized.
The problem appears both in Firefox and Chrome

It is illustrated in the videos below:

https://user-images.githubusercontent.com/113735922/190832056-dba49775-19fd-47cc-a456-b0e3592514c3.mp4

https://user-images.githubusercontent.com/113735922/190832081-a2a15c1e-1d32-447e-a62f-3493f516f18d.mp4

### PR DESCRIPTION
**Pull request by manhcuongincusar1**
## Launch Checklist

Fixes #1654 

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Link to related issues.
 - [x] Include before/after visuals or gifs if this PR includes visual changes.
 - [x] Write tests for all new functionality.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## main
+- Add queryTerrainElevation allows getting terrain elevation in meters at specific point ([#2264](https://github.com/maplibre/maplibre-gl-js/pull/2264))
```
Comment (HarelM) [inline CHANGELOG.md:2]: This and the following item shouldn't be here, they should be under features and improvements... I probably missed that in a previous PR...

**Comment by manhcuongincusar1** [comment]
@HarelM  I created a new pull request for queryTerrainElevation

**Comment by HarelM** [comment]
Can you please close the previous one then? Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (manhcuongincusar1) [reply CHANGELOG.md:2]: okay, let me fix it

Comment (manhcuongincusar1) [reply CHANGELOG.md]: did it

**Comment by manhcuongincusar1** [comment]
@HarelM  I already move my item to "features" section

**Review by HarelM — APPROVED** [review]

**Comment by vignesh882** [comment]
In Mapbox, I can get the elevation of a point using the map.queryTerrainElevation(coordinate) from terrain RGB tile, how can I do the same here?

**Comment by HarelM** [comment]
You'll need to use one of the 3.0 pre-release versions until we release 3.0 officially.

**Pull request merged**